CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
 
 ### Changed
 
+* Fix Style/AccessorMethodName ([#1228](https://github.com/cucumber/cucumber-ruby/pull/1228) [@jaysonesmith](https://github.com/jaysonesmith))
```

Comment:
Updating the Changelog in a pull-request will most likely lead to merge conflict (with Changelog changes in other pull-requests). AFAIK out guideline is that the person the merges a pull-request updates the Changelog, instead of the pull-request author (in the pull-request).

Reply:
I really feel that "Changed" is the wrong heading in the Changelog for these kind of changes. I think "Changed" should mean "different behavior compared to the previous release" - think that a user need to examine carefully before starting to use a new version. Things not affecting the behavior should be placed under other headings like "Refactoring / Developer Experience" as in v3.0.0.pre.2 of "Improved" as in v3.0.0.

Reply:
@brasmusson Unfortunately keepachangelog doesn't seem to indicate any way of dealing with these types of changes. Is this issue only our Ruby repo deals with or does this type of thing happen elsewhere, too? If elsewhere, should we discuss on the monorepo to draw a wider audience?

Ultimately I can definitely move things, it just puts us out of sync with the other repos. Also, once all of this style work is complete, we can enforce the rubocop style set that we end up with so that we won't have any more of these types of PRs, right?